addons/l10n_it/models/account_move.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    def _extend_with_attachments(self, attachments, new=False):
+        return super(AccountMove, self.with_context(
```

Comment:
Thanks to https://github.com/odoo/enterprise/pull/31640 there's no more context key